R/gh.R

```diff
@@ -208,6 +211,24 @@ gh <- function(endpoint, ..., per_page = NULL, .token = NULL, .destfile = NULL,
   res
 }
 
+gh_response_length <- function(res) {
+  if (!is.null(names(res)) && length(res) > 1 &&
+      names(res)[1] == "total_count") {
```

Comment:
Is `total_count` always the first entry or would it be better to just use `%in%`?

Reply:
It seems like it is the first entry, yeah.